web-animations/interfaces/AnimationEffectTiming/easing.html

```diff
@@ -40,17 +40,49 @@
 });
 
 test(function(t) {
+  var invalidEasings = [
+    '',
+    'test',
+    'initial',
+    'inherit',
+    'unset',
+    'steps(3, nowhere)',
+    'steps(-3, end)',
+    'cubic-bezier(0.1, 0, 4, 0.4)',
+    'function (a){return a}',
+    'function (x){return x}',
+    'function(x, y){return 0.3}',
+    '7',
+  ];
   var div = createDiv(t);
-  var anim = div.animate({ opacity: [ 0, 1 ] }, 100 * MS_PER_SEC);
-  assert_throws({ name: 'TypeError' },
-                function() {
-                  anim.effect.timing.easing = '';
-                });
-  assert_throws({ name: 'TypeError' },
-                function() {
-                  anim.effect.timing.easing = 'test';
-                });
-}, 'Test invalid easing value');
+  var anim = div.animate({});
```

Comment:
Do we need the div temporary? Maybe just have
var anim = createDiv(t).animate({});

Also, does it need to have keyframes, or could it be animate(null)?